packages/driver/src/cypress/error_messages.ts

```diff
@@ -141,6 +141,8 @@ export default {
     reserved_word: {
       message: `${cmd('as')} cannot be aliased as: \`{{alias}}\`. This word is reserved.`,
     },
+    invalid_options: `${cmd('as')} only accepts an \`options\` object for its second argument. You passed: \`{{arg}}\``,
```

Comment:
Other error messages for options object  doesn't user back-ticks. should we remove to be consistent?

Reply:
Will do.